components/twitter/actions/advanced-search/advanced-search.mjs

```diff
@@ -0,0 +1,112 @@
+import twitter from "../../twitter.app.mjs";
+
+export default {
+  key: "twitter-advanced-search",
+  name: "Advanced Search",
+  description: "Return Tweets that matches your search criteria.",
+  version: "0.0.3",
+  type: "action",
+  props: {
+    db: "$.service.db",
+    twitter,
+    q: {
+      propDefinition: [
+        twitter,
+        "q",
+      ],
+    },
+    resultType: {
+      propDefinition: [
+        twitter,
+        "resultType",
+      ],
+    },
+    includeRetweets: {
+      propDefinition: [
+        twitter,
+        "includeRetweets",
+      ],
+    },
+    includeReplies: {
+      propDefinition: [
+        twitter,
+        "includeReplies",
+      ],
+    },
+    lang: {
+      propDefinition: [
+        twitter,
+        "lang",
+      ],
+    },
+    locale: {
+      propDefinition: [
+        twitter,
+        "locale",
+      ],
+    },
+    geocode: {
+      propDefinition: [
+        twitter,
+        "geocode",
+      ],
+    },
+    sinceId: {
+      propDefinition: [
+        twitter,
+        "sinceId",
+      ],
+    },
+    enrichTweets: {
+      propDefinition: [
+        twitter,
+        "enrichTweets",
+      ],
+    },
+    count: {
+      propDefinition: [
+        twitter,
+        "count",
+      ],
+    },
+    maxRequests: {
+      propDefinition: [
+        twitter,
+        "maxRequests",
+      ],
+    },
+  },
+  async run() {
+    const {
+      lang,
+      locale,
+      geocode,
+      resultType,
+      enrichTweets,
+      includeReplies,
+      includeRetweets,
+      sinceId,
+      maxRequests,
+      count,
+      q,
+    } = this;
+
+    const limitFirstPage = !sinceId;
+
+    // run paginated search
+    return await this.twitter.paginatedSearch({
```

Comment:
Since this is an async function. [You don't need to `return await`](https://eslint.org/docs/rules/no-return-await). Could you please remove it?